### PR TITLE
T1

### PR DESCRIPTION
**Pull request by kele-leanes**

### CODE REVIEW

**Comment by juanii000** [comment]
You should use a more descriptive name on PR

File: server.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+});
+
+server.post('/productos', [productValidator, allowCors], (req, res) => {
+    productos.push(req.body);
+    console.log('El producto ha sido agregado con exito');
+    console.log(productos);
+    res.json(req.body);
+});
+
+server.get('/productos-favoritos', allowCors, (req, res) => {
```
Comment (juanii000) [inline server.js:165]: You shuldnt use "-" in path name, maybe you can use a new path like /productos/favoritos

File: server.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    const favorites = productos.filter(elem => elem.isFavorite == true);
+    console.log(favorites);
+    res.json(favorites);
+    
+});
+
+server.post('/register', allowCors, (req, res) => {
+    const { mail } = req.body
```
Comment (juanii000) [inline server.js:173]: if it is only one variable you can use const mail = req.body.mail; but is okay on either ways

File: server.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    const validated = newUserVerify(mail);
+    console.log(validated);
```
Comment (juanii000) [inline server.js:175]: To send this code review to master you should delete all of the console.logs that you have, they are only for testing purpose.

File: server.js (hunk 1 above)
Comment (kele-leanes) [reply server.js:165]: i have an issue with this path, because 'productos/favoritos' matches with 'productos/:idproducto'. Do you have any solution?

Comment (juanii000) [reply server.js:163]: yes you need to do the server.get('/productos/favoritos'.... before server.get('/productos/:idproducto so with that you can check fisrt if it is a favorite route and the do the dinamic route if its not

**Pull request merged**